awx/sso/pipeline.py

```diff
@@ -54,7 +54,7 @@ def prevent_inactive_login(backend, details, user=None, *args, **kwargs):
         raise AuthInactive(backend)
 
 
-def _update_m2m_from_expression(user, rel, expr, remove=True):
+def _update_m2m_from_expression(user, rel, expr, remove=True, saml_team_names=False):
```

Comment:
@aperigault did you intend to commit this?  It looks unrelated to the PR subject.

Reply:
Sorry its related to another PR.
Fixed.